### DISCUSSION TITLE
【灌水】测试一下发图&视频（内有纳塔交响音乐320Kbps资源）

### DISCUSSION THREAD

**Discussion by MTAlexKen** [share121/inter-knot#3116] [其他]
# rt
![image](https://github.com/user-attachments/assets/5b62ce5f-86a1-4c95-80c6-7938e7d56849)

---
#### 《原神》纳塔交响音乐现场

https://github.com/user-attachments/assets/83565452-7a01-4315-b485-a8e8a7aa62ce

[直链下载 苑迪萌（HOYO-MiX） - 纳塔 Natlan](https://github.com/MTAlexKen/Vault/blob/main/%E8%8B%91%E8%BF%AA%E8%90%8C%EF%BC%88HOYO-MiX%EF%BC%89%20-%20%E7%BA%B3%E5%A1%94%20Natlan.mp3)

https://github.com/user-attachments/assets/8ea66c29-0d5a-45b4-b4c6-ff33ae0879f8

[直链下载 王予曦（HOYO-MiX） - 苍原的颂灵歌 Anthem of the Savannah.mp3](https://github.com/MTAlexKen/Vault/blob/main/%E7%8E%8B%E4%BA%88%E6%9B%A6%EF%BC%88HOYO-MiX%EF%BC%89%20-%20%E8%8B%8D%E5%8E%9F%E7%9A%84%E9%A2%82%E7%81%B5%E6%AD%8C%20Anthem%20of%20the%20Savannah.mp3)

---
### 为什么你的网站大小应控制在14kB以内
>**[https://jandan.net/p/117485#/](url)**
```
网站小于14kB，加载速度明显提升，关键在于TCP慢启动算法。
```

---
## ToDoList
- [x] 发图
- [x] ToDo
- [x] ~~删除线~~
- [x] **粗体**
- [x] _斜体_
- [x] Markdown语法
- [x] 超链接
- [x] 嵌入视频
- [x] 音乐播放器

**Comment by youyiMC** [comment]
嵌入代码是没法用了的，因为github不让，怕恶意脚本。
但可以试试：
`[![B站视频封面](https://图片地址)](https://www.bilibili.com/video/BVxxxx)`

下面我来实践一下：
[![B站视频封面](https://i0.hdslb.com/bfs/archive/d3285521d27275bca4e28e326b54aa9740dfed32.jpg)](https://www.bilibili.com/video/BV1K4TDzpEvE/)

**Reply by youyiMC** [reply]
点击图片就可以访问视频网页
